### ISSUE TITLE
ExpandableSecation's toggleText can accept ReactNode

### ISSUE THREAD

**Issue by mareklibra** [patternfly/patternfly-design#1171]
## Requesting new features, enhancements, or design changes to PatternFly
- Describe the use cases and requirements of the new feature or enhancement you are requesting.

By having that, the ExpandableSection text (default, expanded, collapsed) can be enriched for an Icon or other React components.

- Explain *why* this feature is needed and/or why these requirements cannot be met using existing components.

In the [ZTPFW](https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable) project we use this component and would like to show processing state (error, warning, success) of various subcomponents.

- Other
I have a PR ready but finding the PF-contribution process too complicated to push it.
In a nutshell, the requested change is about easing `string` to either `ReactNode` or `ReactChild`:
```
export interface ExpandableSectionProps {
...
  toggleText?: React.ReactNode;
  toggleTextExpanded?: React.ReactNode;
  toggleTextCollapsed?: React.ReactNode;
...
}

**Comment by mareklibra** [comment]
@mcarrano Works for me. Thank you. Closing here.
